### DISCUSSION TITLE
Query about parameter RootFx/y/zc and parameter RootFx/y/zb.

### DISCUSSION THREAD

**Discussion by hhuymxu** [OpenFAST/openfast#2237] [Q&A]
In the use of the Elastodyn module in OpenFAST, I have encountered problems with output channels. I cannot figure out the difference between 'RootFx/y/zc' and 'RootFx/y/zb', or in other words, I don't understand the difference between x/y/zc-axis and x/y/zb-axis. Please help me, thanks a lot! 😢
![微信截图_20240524163619](https://github.com/OpenFAST/openfast/assets/113504890/676b8064-207a-46bf-aa07-31bbbeb7c499)

**Comment by jjonkman** [comment]
Dear @hhuymxu,

In these blade-related coordinate systems, "c" refers to the coned coordinate system that is aligned with the pitch axis of a given blade and is fixed within and moves with the hub, and "b" refers to the blade coordinate system that is aligned with the pitch axis of a given blade and is fixed within and moves with the blade.  The difference between the coned and blade coordinate systems is the pitch orientation of the blade.  See Figures 8 and 9 of the Old FAST v6 User's Guide for illustrations: https://openfast.readthedocs.io/en/main/_downloads/d8bd014121d6505cb25cf49bee5eaa80/Old_FAST6_UsersGuide.pdf.

Best regards,

**Reply by hhuymxu** [reply]
Thank you very much jjonkman. After reading usersguide and your reply, I think I have clarified the relationship and difference between the two.
Best regards,
